feature/session/src/main/res/values/strings.xml

```diff
@@ -17,6 +17,7 @@
     <string name="session_detail_resource">Resources</string>
     <string name="session_detail_resource_video">Video</string>
     <string name="session_detail_resource_slide">Slides</string>
+    <string name="session_detail_share_url">https://droidkaigi.jp/2019/en/timetable/%d/</string>
```

Comment:
It's a very small thing 🙇 
Can you delete last `/` ?

Reply:
👀 
We are sorry, examples we provided were only integer but some of session ids are String. 
So this format argument should be `%s` instead.

Reply:
Thank you. I fixed delete last / and change format argument.